src/index.ts

```diff
@@ -2284,11 +2284,16 @@ async function getAllUnclaimedDepositsWithIdxForAddress({
 	chainId,
 	peanutContractVersion,
 	provider = null,
+	claimedOnly = true,
 }: interfaces.IGetAllUnclaimedDepositsWithIdxForAddressParams): Promise<any[]> {
 	if (provider == null) {
 		provider = await getDefaultProvider(chainId)
 	}
 
+	if (peanutContractVersion == 'v4') {
+		console.warn('WARNING: only returning unclaimed deposits for v4 contracts')
+	}
+
```

Comment:
@borcherd what's the point of this warning?

Reply:
Just to make sure that the implementor knows that they will not receive all transactions, but only the unclaimed ones. (since the deposit is deleted after claiming on v4)